### PR TITLE
Harry FiftyFifty Round - first PR (progress check)

### PR DESCRIPTION
**Pull request by hphilpotts**
Submitting PR to get quick progress check please! 

_Work completed so far:_ 
- FiftyFiftyRound page component added
- Routes updated so that page is included when navigating through quiz
- page matches other round pages through use of the same MUI components and custom components
- round title, questions etc. templated out
- first question from round (q.14) mocked up
- q.14 makes use of  `useGuesses` context

_Specific queries for progress check:_
- Have I used `useGuesses` correctly? Can then roll this out to the remaining questions.
- Toggle button added but not formatted. Thoughts on how this should look? Alignment, width, any extra colour etc.?
- Happy with question phrasing? Any changes? Any hints re. what hints I should include?

Thanks

also hope cricket went well

### CODE REVIEW

File: src/pages/FiftyFiftyRound.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+import { Box, Card, CardContent, FormControl, Grid, ToggleButton, ToggleButtonGroup } from '@mui/material'
+
+import { RoundTitle } from '../components/RoundTitle'
+import { QuestionTitle } from '../components/QuestionTitle'
+import { useGuesses } from '../context/GuessContext'
+
+export const FiftyFiftyRound = () => {
+
+    const { guesses, updateGuesses } = useGuesses()
```
Comment (mdnrosen) [inline src/pages/FiftyFiftyRound.jsx]: You don't need 'guesses' at this stage

const { updateGuesses } = useGuesses() is fine

File: src/pages/FiftyFiftyRound.jsx [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+                        <Grid item xs={12}>
+                            <FormControl fullWidth>
+                                <QuestionTitle
+                                    title="True or false"
+                                    num="14"
+                                    question="There will be a super over."
+                                    hint=""
+                                />
+                                <Box alignSelf={'center'}>
+                                    <ToggleButtonGroup
```
Comment (mdnrosen) [inline src/pages/FiftyFiftyRound.jsx:39]: Styling wise how about we try the options to be full width on mobile. Then desktop I'll let you think of something?

File: src/pages/FiftyFiftyRound.jsx [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+                            </FormControl>
+                        </Grid>
+
+                        <Grid item xs={12}>
+                            <FormControl fullWidth>
+                                <QuestionTitle
+                                    title="Pace or spin"
+                                    num="15"
+                                    question="What will take more wickets?"
+                                    hint=""
```
Comment (mdnrosen) [inline src/pages/FiftyFiftyRound.jsx:59]: Leave the hints blank for now - James and I will figure it out

File: src/pages/FiftyFiftyRound.jsx [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+    const { guesses, updateGuesses } = useGuesses()
+
+    const handleChange = (e) => {
+        const { name, value } = e.target
+        updateGuesses(name, value)
+    }
+
+
+    return (
+        <Card sx={{ py: 1 }}>
```
Comment (mdnrosen) [inline src/pages/FiftyFiftyRound.jsx:20]: On your next PR you can swap this for my <RoundCard /> component - I've only just pushed this.

File: src/pages/FiftyFiftyRound.jsx [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+Consider using radio buttons or an even more preferable alternative like:
+-> https://mui.com/material-ui/react-toggle-button/
+
+Two components available:
+
+1. **RoundTitle Component**
+2. **QuestionTitle Component**
+
+The "Round" should be implemented as a <Card /> component. 
+Apply sx={{ my: 2 }} styling to it, similar to how it's done for the "EnglandRound" and "StandingsRound".
```
Comment (mdnrosen) [inline src/pages/FiftyFiftyRound.jsx]: This problem goes away once you implement the RoundCard component

**Review by mdnrosen — APPROVED** [review]

File: src/pages/FiftyFiftyRound.jsx (hunk 4 above)
Comment (hphilpotts) [reply src/pages/FiftyFiftyRound.jsx:20]: Swapping in next PR

**Comment by hphilpotts** [comment]
Merging with main - review comments resolved through minor edit and/or update to TODOs ahead of next PR.

**Pull request merged**